### DISCUSSION TITLE
Contributions not showing in insights

### DISCUSSION THREAD

**Discussion by yudeep-rajbhandari** [community/community#40577] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
I have my repo https://github.com/yudeep-rajbhandari/goBearsBackEnd/ but it does not show my contributions to develop in the insights section as much I have done changes for some reason, the commits are there but the insights contributions are not there.
And the same issue for the repo: https://github.com/PranishBhagatOutlook/se_client
Can you please fix it for let me know what needs to be done to show my actual contributions?

Thanks,
Yudeep

**Comment by ossanna16** [comment]
Hi there @yudeep-rajbhandari and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
